### PR TITLE
ZK-3569: Popup slide-in direction change depending on position relati…

### PR DESCRIPTION
**Pull request by klyvechen**
…ve to input

### CODE REVIEW

File: zul/src/archive/web/js/zul/inp/ComboWidget.js [hunk 1]
```diff
@@ -331,6 +334,14 @@ zul.inp.ComboWidget = zk.$extends(zul.inp.InputWidget, {
 	slideDown_: function (pp) {
 		zk(pp).slideDown(this, {afterAnima: this._afterSlideDown, duration: 100});
 	},
+	/** Slides the drop-up list.
+	 * <p>Default: <code>zk(pp).slideTowardU[(this, {afterAnima: this._afterSlideDown});</code>
+	 * @param DOMElement pp the DOM element of the drop-down list.
+	 * @since 8.5.2
+	 */
+	slideTowardUp_: function (pp) {
```
Comment (wenninghsu) [inline zul/src/archive/web/js/zul/inp/ComboWidget.js]: should think of sth better

**Comment by wenninghsu** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Comment by scribetw** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**